packages/apps/dashboard/server/.gitignore

```diff
@@ -38,4 +38,6 @@ lerna-debug.log*
 /redis_data
 
 .env.development
-.env.production
\ No newline at end of file
+.env.production
+
+e2e.test
```

Comment:
The format of env file name is `.env${NODE_ENV}` and for tests that run via `jest` `NODE_ENV = test` ([ref](https://jestjs.io/docs/environment-variables#node_env)), so

I would suggest to do one of two:
1.
```
.env.test
```
if we want it just automatically

or
2.
```
.env.e2e-test
```
and in `package.json` change the command to be
```
test:e2e": " NODE_ENV=e2e-test jest --config ./test/jest-e2e.json
```